README.md

```diff
@@ -0,0 +1,5 @@
+# Questions about `pull-request`
+ 
+- Is there any example (repository, organization, ...) that shows the use of pull-requests in a classroom?
```

Comment:
You can take a look at https://education.github.community/t/how-to-grade-assignments-on-github/13663
for a few different teacher workflows.